unit/src/RankTwoTensorTest.C

```diff
@@ -187,27 +188,24 @@ TEST_F(RankTwoTensorTest, d2secondInvariant)
   mep = _m3;
   d1 = _m3.dsecondInvariant();
   d2 = _m3.d2secondInvariant();
-  for (unsigned i = 0; i < 3; i++)
-    for (unsigned j = 0; j < 3; j++)
-    {
-      for (unsigned int k = 0; k < 3; k++)
-        for (unsigned int l = 0; l < 3; l++)
+  for (auto i : make_range(3))
```

Comment:
These tests of course would all fail if LIBMESH_DIM were anything other than 3.

Reply:
agree